### ISSUE TITLE
PCPartPicker list

### ISSUE THREAD

**Issue by NFLD99** [Aliucord/plugin-requests#523]
### Plugin Description

This plugin would make a list from a PCPartPicker link, this list would include compatibly,  Estimated Wattage, parts and total cost. Something like the PCPartPicker bot in their discord (screenshot attached below) 

### Go into more detail...

![Screenshot_20220112-114304](https://user-images.githubusercontent.com/24623601/149167558-53726b81-6919-4e32-86fa-ffb95d172e11.png)

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by l6t9** [comment]
Just make your own bot or use an existing one. This isnt really something worth doing as an Aliucord plugin of all things

**Comment by Ushie** [comment]
yes, with userbots existing the development time is better spent with that rather than this
